### ISSUE TITLE
feature request: add attendee to event with mail address

### ISSUE THREAD

**Issue by Nachtlichtermeer** [Intermesh/groupoffice#1125]
version: 6.8.36

when i try to add an attendee to an event in the web GUI i cannot insert a mail address
i only can choose internal users or entries of my address book so i have to create a contact first

through CalDAV all works as expected

**Comment by mschering** [comment]
Yes we are aware you can only add contacts. A new calendar is in the works where you can add any e-mail.

**Comment by dehart** [comment]
In GroupOffice 25.0 attendees can be added by typing a valid email address and press enter in the autocomplete field that also finds contacts and users.

**Comment by dehart** [comment]
duplicate of #616
